bundles/framework/divmanazer/resources/scss/divman.scss

```diff
@@ -317,6 +317,7 @@ body {
         border: 0;
         overflow: auto;
         max-height: 500px;
+        min-width: 580px;
```

Comment:
I can see this causing confusion in the future because .oskari-attached has lower min-width BUT is the parent element of .oskari-flyoutcontentcontainer. So the child element would have larger min-width than the parent. I'd say we need to find another way of solving this.

Reply:
Fixed this with other suggested approach.